requirements.txt

```diff
@@ -1,7 +1,4 @@
-botocore
 Flask==1.1.1
 pytest==5.4.1
 requests==2.23.0
-aws==0.2.5
```

Comment:
Are you sure the aws package is required for the application execution?

Reply:
I checked that. Surprisingly 'aws' import is obsolete. It is not being used anywhere in the project.

Reply:
The aws package is not needed for our app deploy, I was able to deploy the app fully without this requirement. 
I will delete this line and commit a new CR.